orca-clouddriver/src/test/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/WaitForCapacityMatchTaskSpec.groovy

```diff
@@ -49,7 +49,7 @@ class WaitForCapacityMatchTaskSpec extends Specification {
       oort = Stub(OortService)
       oort.getServerGroup("test", "us-east-1", "kato-main-v000") >> { new Response('kato', 200, 'ok', [], new TypedString(mapper.writeValueAsString(serverGroup))) }
       task.oortService = oort
-      def context = [account: "test", "deploy.server.groups": ["us-east-1": ["kato-main-v000"]]]
+      def context = [account: "test", "deploy.server.groups": ["us-east-1": ["kato-main-v000"]], capacity: [desired: 3]]
```

Comment:
Are these changes still needed with the above null check?

Reply:
I will check